vega/utils.py

```diff
@@ -5,18 +5,18 @@
 import copy
 
 
-def update(d, u, overwrite=True):
+def update(d, u):
     """Update dictionary.
 
     If overwrite is False, then existing keys in d won't be overwritten
     by values in u.
     """
     for k, v in u.items():
         if isinstance(v, collections.Mapping):
-            r = update(d.get(k, {}), v, overwrite=overwrite)
+            r = update(d.get(k, {}), v)
             d[k] = r
         else:
-            if overwrite or k not in d:
+            if k not in d:
                 d[k] = u[k]
     return d
```

Comment:
This looks good. Minor points: no need to define ``r``: just do ``d[k] = update(...``.
Also, the ``else: if:`` could just be an ``elif:``

Finally, to be more clear what this function does, perhaps call it ``nested_update``, and change the docstring to be clear that ``d`` is updated in-place with values from ``u`` without modifying existing values.